tests/integration_tests/modules/test_cli.py

```diff
@@ -92,9 +89,3 @@ def test_invalid_userdata_schema(client: IntegrationInstance):
     )
     assert warning in log
     assert "asdfasdf" not in log
-
```

Comment:
This is a dropped duplicate (and incorrect) assertion that is already [correctly checked as exit 2 a couple lines above](https://github.com/canonical/cloud-init/blob/c3a1c8ab1df584de102aff30e08d7ef180bd30d1/tests/integration_tests/modules/test_cli.py#L81-L82)